modules/order/src/Entity/LineItem.php

```diff
@@ -102,7 +102,7 @@ public function setTitle($title) {
    * {@inheritdoc}
    */
   public function getQuantity() {
-    return $this->get('quantity')->value;
+    return (string) $this->get('quantity')->value;
```

Comment:
It would be more proper to change a field type, but that will probably require providing a custom widget. We can go with the cast and open a followup issue.